core/new-gui/src/app/home/component/home.component.spec.ts

```diff
@@ -22,8 +22,4 @@ describe("HomeComponent", () => {
     component = fixture.componentInstance;
     fixture.detectChanges();
   });
-
-  it("should create", () => {
-    expect(component).toBeTruthy();
-  });
```

Comment:
why removing this test? each component at minimal should have a creation test.

Reply:
Fixed.